Apps/Admin/Tests/Functional/cypress/integration/ui/support.cy.js

```diff
@@ -0,0 +1,298 @@
+        cy.get("[data-testid=query-type]").contains("HDID").click();
+        cy.get("[data-testid=query-input]").clear().type(hdidNotFound);
+        cy.get("[data-testid=search-btn]").click();
+        cy.get("[data-testid=banner-feedback-error-message]").should(
+            "be.visible"
+        );
+        cy.get("[data-testid=message-verification-table]").should("not.exist");
+
+        // Close banner
+        cy.get("[data-testid=banner-feedback-error-message]").within(() => {
+            cy.get("button").parent(".mud-alert-close").click();
+        });
+
+        cy.log("Verify hdid returns no results test finished.");
+    });
+
+    it("Verify no results sms query.", () => {
+        cy.log("Verify sms returns no results test started.");
+
+        // Search with SMS not found.
+        cy.get("[data-testid=query-type-select]").click();
+        cy.get("[data-testid=query-type]").contains("SMS").click();
+        cy.get("[data-testid=query-input]").clear().type(smsNotFound);
+        cy.get("[data-testid=search-btn]").click();
+        cy.get("[data-testid=message-verification-table]")
+            .find("tr")
+            .should("have.length", 1);
+
+        cy.log("Verify sms returns no results test finished.");
+    });
+
+    it("Verify no results email query.", () => {
+        cy.log("Verify email returns no results test started.");
+
+        // Search with Email not found.
+        cy.get("[data-testid=query-type-select]").click();
+        cy.get("[data-testid=query-type]").contains("Email").click();
+        cy.get("[data-testid=query-input]").clear().type(emailNotFound);
+        cy.get("[data-testid=search-btn]").click();
+        cy.get("[data-testid=message-verification-table]")
+            .find("tr")
+            .should("have.length", 1);
+
+        cy.log("Verify email returns no results test finished.");
+    });
+
+    it("Verify invalid phn.", () => {
+        cy.log("Verify invalid phn test started.");
+
+        // Search by invalid PHN.
+        cy.get("[data-testid=query-type-select]").click();
+        cy.get("[data-testid=query-type]").contains("PHN").click();
+        cy.get("[data-testid=query-input]").clear().type(phnInvalid);
+        cy.get("[data-testid=search-btn]").click();
+        cy.get(".d-flex").contains("Invalid PHN").should("be.visible");
+        cy.get("[data-testid=message-verification-table]").should("not.exist");
+
+        cy.log("Verify invalid phn test finished.");
+    });
+
+    it("Verify required paramater.", () => {
+        cy.log("Verify required parameter test started.");
+
+        // Search by empty PHN.
+        cy.get("[data-testid=query-type-select]").click();
+        cy.get("[data-testid=query-type]").contains("PHN").click();
+        cy.get("[data-testid=query-input]").clear();
+        cy.get("[data-testid=search-btn]").click();
+        cy.get(".d-flex")
+            .contains("Search parameter is required")
+            .should("be.visible");
```

Comment:
Nit: `.d-flex` is not a great target for tests because it's used for styling purposes in a lot of places.  In the future, we should try narrowing the scope by targeting a more specific element and using the `parents()`, `siblings()`, or `children()` commands to get to the element we're interested in.